apps/charge-accounts-service/src/operators/operators.service.ts

```diff
@@ -67,12 +67,15 @@ export class OperatorsService {
   }
 
   async createWallet (createOperatorWalletDto: CreateOperatorWalletDto): Promise<OperatorWallet> {
-    const createdOperatorWallet = new this.operatorWalletModel(createOperatorWalletDto)
+    const createdOperatorWallet = new this.operatorWalletModel({
+      ...createOperatorWalletDto,
```

Comment:
@LiorAgnin instead of destructuring it here, we can do the toLowerCase in controller:
https://github.com/fuseio/fusebox-backend/blob/32ea2032fb31d94c1a8159a3df32c573c90368d5/apps/charge-accounts-service/src/operators/operators.controller.ts#L118